### DISCUSSION TITLE
Moving the CLI to a separate repository

### DISCUSSION THREAD

**Discussion by paganotoni** [gobuffalo/buffalo#2130] [Ideas]
While wanting to use `embed` instead of Packr in Buffalo one of the things I've noticed is that the CLI may change by different reasons than the Buffalo library.  I was wondering if its a good time to move the CLI to a separate repo so its maintenance and releases are more wired to the way things happen in the CLI rather than the way things happen on the library.

cc @gobuffalo/core-managers @markbates

**Comment by fasmat** [comment]
Makes sense imho

**Comment by paganotoni** [comment]
Next version of the CLI would be v0.1.0 and we would release v0.17.0 version of the library as this would be a breaking change.

**Comment by paganotoni** [comment]
Update on the CLI extraction topic:
- I went ahead and copied the code as it is into the gobuffalo/cli repo seems I was able to setup homebrew for the v0.17.0-beta thats been cooked there.

- I have a branch for a new version of the library that does not contain CLI packages except for plugins  which I'm hoping to redo later on the road but for now there is a dependency to it in the Buffalo library.

- I created [this project](https://github.com/gobuffalo/cli/projects/1) To give a bit of visibility to the CLI extraction tasks.

**Comment by paganotoni** [comment]
Update:
- Released Buffalo v0.17.2 (had a couple of Go mod learnings here)
- Released cli v0.17.1
- Updated documentation to point the cli on the installation instruction
- Updated release script to keep current installation as it is.

We can consider this one done.
